src/Traits/MutatesMigrationCommands.php

```diff
@@ -72,8 +72,8 @@ protected function getMigrationPaths()
         }
 
         // Tenant migrations path is configured.
-        if ($path = config('tenancy.db.tenant-migrations-path')) {
-            return [$path];
+        if ($path = (array) config('tenancy.db.tenant-migrations-path')) {
```

Comment:
A little test in tinker:


```
>>> if ($var = (array) 'bar') { echo $var; }
ArrayPHP Notice:  Array to string conversion in Psy Shell code on line 2
>>> if ($var = (array) null) { echo $var; }
>>> if ($var = (array) '') { echo $var; }
ArrayPHP Notice:  Array to string conversion in Psy Shell code on line 2
>>>
```

Aka this might not be the best way to do the conditional.